op-node/rollup/derive/channel_out.go

```diff
@@ -107,31 +106,27 @@ func (co *SingularChannelOut) Reset() error {
 	return err
 }
 
-// AddBlock adds a block to the channel. It returns the RLP encoded byte size
+// AddBlock adds a block to the channel. It returns the block's L1BlockInfo
```

Comment:
Looks like we were never returning the RLP encoded byte size before, contrary to the godoc.